src/containers/profile-page/store/lineups/tracks/sagas.js

```diff
@@ -43,14 +43,17 @@ function* getTracks({ offset, limit, payload }) {
   const sort = payload.sort === TracksSortMode.POPULAR ? 'plays' : 'date'
 
   if (user._artist_pick) {
+    const getUnlisted = true
```

Comment:
some users may not have an artist pick, should we make the change on line 93 below?
```
const currentUser = yield select(getAccountUser)
if (currentUser.handle.toLowerCase() === handle.toLowercase()) {
  getUnlisted = true
}
```

Reply:
Yes that was an oversight on my part, good catch!